utilities/include/structs.h

```diff
@@ -229,7 +230,13 @@ struct Curvature {
  * Mesh.
  */
 struct BaryRef {
-  /// Reference to original Mesh, matching Manifold.GetMeshIDs().
+  /// Identifier of the triangle, can be mapped to the original mesh ID.
+  /// - In `MeshRelation`: The original mesh triangle index.
+  /// - In `MeshRelationD`: The original mesh triangle index =
+  /// `originalID[meshID]`
+  ///
+  /// @note Triangles coming from different manifolds should have different mesh
+  /// ID, otherwise `SimplifyTopology` will not work properly.
```

Comment:
This is public API documentation (parsed by Doxygen) so it shouldn't refer to anything internal like `MeshRelationD` or `SimplifyTopology`. Let's move these comments to the `Impl`.